src/bindgen/library.rs

```diff
@@ -443,4 +450,88 @@ impl Library {
             x.mangle_paths(&monomorphs);
         }
     }
+
+    // Replace all associated types with concrete types
+    fn replace_assoc_types(&mut self) {
```

Comment:
I told you already to also replace type inside `GenericParamType`

Reply:
Unions, Structs, Enums, Typedefs have it. I have implemented it.

Reply:
I don't see in code that you have replaced `generic_params` of any of those